mllib/src/main/scala/org/apache/spark/ml/param/params.scala

```diff
@@ -55,58 +49,42 @@ class Param[T] (
    */
   def ->(value: T): ParamPair[T] = ParamPair(this, value)
 
-  override def toString: String = {
-    if (defaultValue.isDefined) {
-      s"$name: $doc (default: ${defaultValue.get})"
-    } else {
-      s"$name: $doc"
-    }
-  }
+  override def toString: String = s"$name: $doc"
```

Comment:
I thought about this option but I'm not sure whether we should put default value and current value in `Param.toString` because they are stored in the parent. But no strong preference.